src/modules/faqs/faqs.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable, BadRequestException, ForbiddenException, InternalServerErrorException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Faqs } from './entities/faqs.entity';
+import { CreateFaqDto } from './dto/createFaqsDto';
+import { User } from '../user/entities/user.entity';
+
+@Injectable()
+export class FaqsService {
+  constructor(
+    @InjectRepository(Faqs) private readonly faqsRepo: Repository<Faqs>,
+    @InjectRepository(User) private readonly userRepo: Repository<User>
+  ) {}
+
+  async createFaq(createFaqs: CreateFaqDto, userId: string) {
+    try {
+      const user = await this.userRepo.findOneBy({ id: userId });
```

Comment:
Don't re invent the wheel. There is a method in the User service to get a user, use that

Reply:
@Homoakin619 The methods are private for the user service, can I update it?

Reply:
@Demandtech You're only trying to find the user by id, nothing more that. So get it from the User service.

Reply:
there is a method in the UserService "getUserRecord" use that

Reply:
But user_type was not added to the type interface... @Homoakin619